understanding/20/focus-order.html

```diff
@@ -55,6 +53,28 @@ <h2>Intent of Focus Order</h2>
          sets of users and does not appear to either of them to jump around randomly. 
       </p>
       
+      <p>Focus order also does not need to follow the visual presentation of the web page,
+         as long as the order in which elements receive focus is logical, and the hierarchy and relationship
+         of content implied by the visual presentation is preserved. For instance, in a two-column
+         layout, the assumption in western left-to-right systems may be that elements in the
+         left-hand column would receive focus first – and, more generally, that focus order proceeds across lines,
+         from components in the top-left corner of the viewport to the bottom-right corner.
+         However, if meaning and operation are not affected, in this case it is not a failure if elements in the right-hand
+         column receive focus first. As a best practice, though, make sure the focus order reinforces the reading order.
+      </p>
+      
+      <div class="note">
+         <p>It is not a failure of this success criterion if static/non-interactive content
+         receives focus. Making an element that does not act as an actionable user interface component
+         focusable, or programmatically moving focus to such an element, does not necessarily constitute
+         a failure of this success criterion. This situation is a failure if these additional focus stops
+         impede the operation of the content, or create illogical focus orders - for example, a control
+         appearing to receive focus multiple times due to the use of nested focusable elements.</p>
+         <pre><code>&lt;div tabindex="0"&gt;&lt;button&gt;...&lt;/button&gt;&lt;/div&gt;</code></pre>
+         <p>As a best practice, avoid having focusable elements which cannot be operated
+         or actioned, as these are likely to make operation unnecessarily tedious for keyboard users.</p>
```

Comment:
Recommend removing "unnecessarily" - is tedious ever a design decision borne out of necessity?

Reply:
yup, good catch